scripts/build_fhe_c_test.zsh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env zsh
+
+set -ex
+
+disable -r time
+
+# cargo build --release --features c --example circ 
+
+BIN=./target/release/examples/circ
+export CARGO_MANIFEST_DIR=$(pwd)
+
+case "$OSTYPE" in 
+    darwin*)
+        alias measure_time="gtime --format='%e seconds %M kB'"
+    ;;
+    linux*)
+        alias measure_time="time --format='%e seconds %M kB'"
+    ;;
+esac
+
+function fhe_test {
+    cpath=$1
+    RUST_BACKTRACE=1 measure_time $BIN $cpath fhe
+}
+
+# build boolean tests
+fhe_test ./examples/C/fhe/unit_tests/boolean_tests/2pc_boolean_and.c
+fhe_test ./examples/C/fhe/unit_tests/boolean_tests/2pc_boolean_or.c
```

Comment:
I think we can add the xor test here as well.

Reply:
There is no boolean xor operator in C so I am not sure how to write the test.